source/libspirv.cpp

```diff
@@ -13,6 +13,7 @@
 // limitations under the License.
 
 #include "spirv-tools/libspirv.hpp"
+#include <iostream>
```

Comment:
This should go above the libspirv.hpp header

Reply:
Forgot to delete.